examples/running_flct.py

```diff
@@ -0,0 +1,71 @@
+"""
+===================================
+Fourier Local Correlation Tracking
+===================================
+
+This example applies Fourier Local Correlation Tracking (FLCT)
+to a set of two images taken within a short interval of each other
+using `sunkit_image.flct.flct <http://solarmuri.ssl.berkeley.edu/~fisher/public/software/FLCT/C_VERSIONS/>`__.
+"""
+
+import numpy as np
+
+import sunkit_image.flct as flct
+import matplotlib.pyplot as plt
+
+###########################################################################
+# This examples describes the objective and the probable use case to find the
+# 2D velocity flow field between two images taken within a short span of time.
+# So to demonstrate that we will create two dummy images containing a moving
+# object and assume the time difference between the images to be one second.
+
+# Creating the input arrays
+image1 = np.zeros((10, 10))
+image1[0:3, 0:3] = 1
+
+image2 = np.zeros((10, 10))
+image2[1:4, 1:4] = 1
+
+# Plot both the images
+fig = plt.figure(figsize=(1, 2))
+
+ax1 = fig.add_subplot(121)
+plt.imshow(image1)
+
+ax2 = fig.add_subplot(122)
+plt.imshow(image2)
+
+###########################################################################
+# Now we come to the main function where FLCT is applied. We will pass the
+# above arrays to `sunkit_image.flct.flct`
+
+# Since the input arrays were stored in a row major format, so no swapping
+# needs to take place. The values of the parameters were used, were the ones
+# which gave the best visual result of the velocities.
+vel_x, vel_y, vm = flct.flct(image1, image2, 1, 1, 2.3)
+
+###########################################################################
+# The return values are the two dimensional velocity field with `vel_x`
+# showing velocity in the x-direction and `vel_y`, the velocity in the
+# y-direction. The last, ``vm`` is the mask array which shows the pixel
+# locations where the FLCT calculations were done.
+
+# We will also plot the velocity in both 'x' and 'y' directions
+fig = plt.figure(figsize=(1, 2))
+
+ax1 = fig.add_subplot(121)
+plt.imshow(vel_x)
+
+ax2 = fig.add_subplot(122)
+plt.imshow(vel_y)
+
+################################################################################
+# You must have noticed that both the velocities in ``x`` and ``y`` have similar
+# plots with most of the pixel values having values equal to 1. This denotes that
+# each pixel has a velocity of 1 in both the direction. So net velocity of each
+# velocity is diagonal which should be the case as evident from the two input images.
+# Notice the lonely pixel which has value less than zero. This discrepancy is due
+# one of the limitations of the `flct` algorithm.
+
+# Show all the plots
+plt.show()
```

Comment:
I think you want to break this example up further. 

Start image -> move to the left by 1 -> move down by 1 -> move down and left by 1 at the same time.

Just to show the results and that it works.